pkg/webui/console/views/application-integrations-lora-cloud/index.js

```diff
@@ -0,0 +1,115 @@
+// Copyright © 2020 The Things Network Foundation, The Things Industries B.V.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import React from 'react'
+import { Container, Col, Row } from 'react-grid-system'
+import { defineMessages } from 'react-intl'
+import { useSelector } from 'react-redux'
+
+import LoRaCloudImage from '@assets/misc/lora-cloud.png'
+
+import PageTitle from '@ttn-lw/components/page-title'
+import Breadcrumb from '@ttn-lw/components/breadcrumbs/breadcrumb'
+import { withBreadcrumb } from '@ttn-lw/components/breadcrumbs/context'
+import Link from '@ttn-lw/components/link'
+
+import Message from '@ttn-lw/lib/components/message'
+import ErrorView from '@ttn-lw/lib/components/error-view'
+
+import LoRaCloudForm from '@console/containers/lora-cloud-form'
+
+import Require from '@console/lib/components/require'
+
+import SubViewError from '@console/views/error/sub-view'
+
+import sharedMessages from '@ttn-lw/lib/shared-messages'
+
+import { mayViewOrEditApplicationPackages } from '@console/lib/feature-checks'
+
+import { selectSelectedApplicationId } from '@console/store/selectors/applications'
+
+import style from './application-integrations-lora-cloud.styl'
+
+const m = defineMessages({
+  loRaCloudDeviceManagement: 'LoRa Cloud Device Management',
+  officialLoRaCloudDocumentation: 'Official LoRa Cloud documentation',
+  loRaCloudInfoText: `With the LoRa Cloud Device & Application Services protocol, you can manage common device functionality at the application layer for LoRaWAN®-enabled devices. This protocol consists of a set of messages that are exchanged on a predefined device management LoRaWAN port. The purpose of these messages is three-fold:
+<ol><li>Periodically communicate info messages</li><li>Trigger client-initiated management commands</li><li>Run advanced, application-layer protocols which solve common LoRaWAN use cases</li></ol>`,
```

Comment:
`on a predefined device management LoRaWAN port.` maybe there should be a reference that this defaults to `199`.

`on a predefined device management LoRaWAN port. By default this port is 199.` perhaps ?